web/source/resources/osk/kmwosk.css

```diff
@@ -139,7 +139,8 @@
 .kmw-title-bar-actions{position: absolute; cursor: default; right:2px; top:0; width: 33px; height: 13px;}
 .kmw-footer-caption{color:#fff;font:0.7em Arial;margin:0 0 0 4px;}
 
-.kmw-banner-bar{height:40px; width:auto; margin:0; background-color:lightblue; border-radius:0 0 6px 6px; }
+.kmw-banner-bar{height:40px; width:100%; margin:0; background-color:lightblue; border-radius:0 0 6px 6px; }
+.kmw-suggestion-text{display: inline-block; text-align:center; width: 33%;}
```

Comment:
The styling needs some additional work before this gets merged in - we have very analogous styling already in this .css file, so it's mostly a matter of replicating existing styles as we might prefer.

`.kmw-banner-bar`'s 'background-color setting should be set similar to the following:
* `.phone.ios .kmw-key-layer-group {background-color: #cfd3d9}`
* `.phone.android .kmw-key-layer-group {background-color: #333;}`
* As for desktop, I'd imagine either using the same `.kmw-key-layer-group` or using the outer orange would be best.

Regarding `.kmw-suggestion-text`:
* `.phone.ios .kmw-key.kmw-key-default {color:#000;background-color:#fdfdfe;} `
* `.phone.android .kmw-key.kmw-key-default {color:#fff;background-color:#777;}`
* `.kmw-key-default{color:#000;background-color:#eee;}` // Desktop

Reply:
here's updated screenshots

### iOS
![new ios suggestion](https://user-images.githubusercontent.com/7358010/54738871-91ac6100-4be8-11e9-83d9-d46395621de5.png)

### android
![new android suggestions](https://user-images.githubusercontent.com/7358010/54738874-97a24200-4be8-11e9-9fc0-eb68d14c9468.png)


### desktop (not centered but I think we can defer fixing this and focus on embedded apps)
![new desktop suggestions](https://user-images.githubusercontent.com/7358010/54738900-ae489900-4be8-11e9-8482-e13633931ced.png)

Reply:
Looks close enough to work for an initial take, though obviously it could use a bit more styling eventually.